packages/api/server/channels/app.mts

```diff
@@ -50,39 +50,34 @@ async function previewStart(
   const existingProcess = processMetadata.get(app.externalId);
 
   if (existingProcess) {
-    conn.reply(`app:${app.externalId}`, 'preview:status', {
+    wss.broadcast(`app:${app.externalId}`, 'preview:status', {
```

Comment:
I realize this is sorta confusing, but the `conn` object represents one specific connection and thus `conn.reply` only replies to that one connected client, whereas `wss.broadcast` broadcasts to all clients connected over a given topic.

One of the problems this can cause here is that, since this is being held in a closure for the duration of a long-lived process, the connection may close long before the preview server is done running (refreshing the page will cause this). After the connection is closed, all preview server output would never make its way to any client.

cc @1egoman @nichochar